src/Native/pkg/runtime.native.System.Net.Http/runtime.native.System.Net.Http.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   
   <PropertyGroup>
-    <Version>4.0.1</Version>
+    <Version>4.0.2</Version>
```

Comment:
Are you sure we actually need to update the shim?  /cc @davidsh 

If we do then we'll also need to ship an updated lineup package, since that's what pulls in the runtime-specific shim packages.